languages/elixir/reference/exercise-concepts/secret-handshake.md

```diff
@@ -0,0 +1,53 @@
+# Concepts for secret-handshake
+
+## Optimal solution
+```elixir
+defmodule SecretHandshake do
```

Comment:
I think "modules" are a Concept too?

Reply:
They are, we have Modules reflected in #855, but good to reflect that here too

Reply:
Okay good!